odoo/http.py

```diff
@@ -904,8 +904,7 @@ class EndPoint(object):
     def __init__(self, method, routing):
         self.method = method
         self.original = getattr(method, 'original_func', method)
-        self.routing = routing
-        self.arguments = {}
```

Comment:
Was it problematic to keep it? Removing it in 13.0 just because it is unused is probably not worth but since it was discussed and validated in the discord :+1: